src/hardware/vga_draw.cpp

```diff
@@ -344,38 +344,6 @@ static uint8_t * VGA_Draw_Linear_Line(Bitu vidstart, Bitu /*line*/) {
 	return ret;
 }
 
-static uint8_t *VGA_Draw_Xlat16_Linear_Line(Bitu vidstart, Bitu /*line*/)
-{
-	const auto offset = vidstart & vga.draw.linear_mask;
-	const uint8_t *ret = &vga.draw.linear_base[offset];
-	const uint16_t *xlat16 = vga.dac.xlat16;
-
-	// see VGA_Draw_Linear_Line
-	if (GCC_UNLIKELY((vga.draw.line_length + offset)& ~vga.draw.linear_mask)) {
-		const auto end = (offset + vga.draw.line_length) &
-		                 vga.draw.linear_mask;
-
-		// assuming lines not longer than 4096 pixels
-		const auto wrapped_len = end & 0xFFF;
-		const auto unwrapped_len = vga.draw.line_length - wrapped_len;
-
-		// unwrapped chunk: to top of memory block
-		for (uintptr_t i = 0; i < unwrapped_len; ++i)
-			write_unaligned_uint16_at(TempLine, i, xlat16[ret[i]]);
-
-		// wrapped chunk: from base of memory block
-		for (uintptr_t i = 0; i < wrapped_len; ++i)
-			write_unaligned_uint16_at(TempLine, i + unwrapped_len,
-			                          xlat16[vga.draw.linear_base[i]]);
-
-	} else {
-		for (uintptr_t i = 0; i < vga.draw.line_length; ++i) {
-			write_unaligned_uint16_at(TempLine, i, xlat16[ret[i]]);
-		}
-	}
-	return TempLine;
-}
-
```

Comment:
Ok this was the 👾 🐷 business if I'm not mistaken. Seems very specific for a single obscure demo.